core-web/apps/dotcms-ui/src/stories/primeng/media/Image.stories.ts

```diff
@@ -0,0 +1,32 @@
+import { Meta, Story } from '@storybook/angular';
+
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+
+import { ImageModule } from 'primeng/image';
+
+export default {
+    title: 'PrimeNG/Media/Image',
+    component: Image,
+    args: { src: 'https://picsum.photos/600/900', alt: 'Image Description', preview: true },
+    parameters: {
+        docs: {
+            description: {
+                component: 'Displays an image with preview and tranformation options.'
+            }
+        }
+    }
+} as Meta;
+
+const Template = `<p-image [src]="src" [alt]="alt" [preview]="preview"></p-image>`;
+
+export const Main: Story = (args) => {
```

Comment:
`Story` is deprecated, can we check what's the new thing? so we don't use deprecated things in new stories.

Reply:
moved to `StoryFn`